csu

```diff
@@ -176,7 +176,7 @@ function static_scratch {
 # Collecting static files
 function collect_static {
   printf "\nCollecting static files..."
-  docker-compose exec django /docker_venv/bin/python3 ./manage.py collectstatic --no-input
+  docker-compose exec django /docker_venv/bin/python3 ./manage.py collectstatic --no-input --clear
```

Comment:
See: https://docs.djangoproject.com/en/1.11/ref/contrib/staticfiles/#cmdoption-collectstatic-clear

We wipe the `build` folder on each build of the static files, we probably should be wiping `staticfiles` too!